app/models/relay_state.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require 'sentry_logging'
+
+class RelayState
```

Comment:
should this be in lib since it's not being saved anywhere?

Reply:
Sounds ok to me. I've always been a little hazy on which classes belong in /models vs /lib.